remix_01.py

```diff
@@ -34,13 +30,14 @@ def main():
 def filter_json(id):
     url = "http://ccmixter.org/api/query?f=json&t=info&ids="+id
     site = urllib2.urlopen(url).read()
-    data = json.loads(site)
-
-    user_name = data[0]['user_name']
-    bpm = data[0]['upload_extra']['bpm']
-    upload_name = data[0]['upload_name']
-    remix_parents = data[0]['remix_parents'][0]['upload_id']
-    remix_children = data[0]['remix_children'][0]['upload_id']
+    alldata = json.loads(site)
```

Comment:
Because the rest of the code is always using only the first element of `data`, like `data[0]`, why not make that the value of the variable